prometheus/push/example_add_from_gatherer_test.go

```diff
@@ -67,18 +71,16 @@ func ExampleAddFromGatherer() {
 	if err != nil {
 		fmt.Println("DB backup failed:", err)
 	} else {
-		// Only now register successTime.
-		registry.MustRegister(successTime)
+		// Add succesTime to pusher only in case of success.
+		// We could as well register it with the registry.
+		// This example, however, demonstrates that you can
+		// mix Gatherers and Collectors when handling a Pusher.
+		pusher.Collector(successTime)
```

Comment:
Its not clear to me at first glance exactly what this call is doing, is it pushing or is it registering?
`AddCollector` may be clearer

Reply:
I had considered that name, too, but in the end I decided to go for the approach to use a verb only for the methods that are actually performing the push to the PGW (Push/Add) and use nouns for the  methods that configure the push. This also makes the methods shorter. Furthermore, using the verb "add" specifically will easily get confused with the `Add` method.

Hence, I do thing the current naming is the clearest and easiest to handle. Does somebody else have an opinion on this?